test/land/letkfoi_snowda.sh

```diff
@@ -19,11 +19,12 @@ WORKDIR=$project_binary_dir/test/land/letkfoi_snowda
 RSTDIR=$GDASAPP_TESTDATA/lowres/gdas.$GYMD/$GHR/model_data/atmos/restart
 export OBSDIR=$GDASAPP_TESTDATA/land
 
-GFSv17=${GFSv17:-"NO"}
+GFSv17=${GFSv17:-"YES"}
 DAtype=letkfoi_snow
 
 if [ $GFSv17 == "YES" ]; then
     SNOWDEPTHVAR="snodl"
+    cp ${project_source_dir}/ush/land/gfs-land_gfsv17.yaml ${project_binary_dir}/fv3-jedi/test/Data/fieldmetadata/gfs-land.yaml
```

Comment:
where is the `gfs-land.yaml` for non-gfsv17?
This if block is unbalanced

Reply:
`gfs-land.yaml` is a meta data override file in the JEDI fv3-jedi repo. For the GFSv17, the state variable `snodl` defined in the `gfs-land_gfsv17.yaml` is required for the snow DA for handling the fractional grid in GFSv17. However, this `gfs-land_gfsv17.yaml` is not available in the fv3-jedi repo. 
We just had a meeting and plan to replace current `snwdph` with `snodl` in JEDI fv3-jedi repo directly for handling the fractional grid issue. Therefore, we don't need to do any changes in both GDASApp and Global-Workflow. I am going to close these PRs.